docs/guides/conf.py

```diff
@@ -21,16 +21,10 @@
 # can be successfully imported
 sys.path.insert(0, root)
 sys.path.append(root / "docs/guides")
-sys.path.append(root / "cms/djangoapps")
```

Comment:
I am not sure why the changes in this file could be made now, but could not have been made in the past? I guess I don't know exactly what Sphinx is doing here.

Reply:
@sarina So the general "import shims / sys-path-hack removal" initiative has existed to remove edx-platform's special-snowflake way of appending lms/djangoapps, cms/djangoapps, and common/djangoapps into sys.path (that is, the root namespace from which one can import Python modules).

This was done with good intentions -- ie, _"pulling `student` into its own repo one day will be easier if we import it as `import student` instead of `import common.djangoapps.student`"_ -- but the dream was never realized. Instead, we just have a bunch of cruft and complexity to handle the weird fact that edx-platform has ~40 importable local packages in the root namespace. 

This is some of that cruft, which I'm removing now, since we no longer treat the djangoapps folders as import roots (as of this PR).

Reply:
@sarina Sorry, I think I mis-interpreted your question.

I could have made these changes a couple months ago when I first removed all instances of the old imports in edx-platform.

I am making them now only because I didn't notice them earlier :)

Reply:
Yes! That second part answered my rambly question. 👍